src/signal.rs

```diff
@@ -107,21 +120,23 @@ extern "C" {
     fn __libc_current_sigrtmax() -> c_int;
 }
 
-/// Returns the minimum (inclusive) real-time signal number.
+/// Return the minimum (inclusive) real-time signal number.
 #[allow(non_snake_case)]
 fn SIGRTMIN() -> c_int {
     unsafe { __libc_current_sigrtmin() }
 }
 
-/// Returns the maximum (inclusive) real-time signal number.
+/// Return the maximum (inclusive) real-time signal number.
 #[allow(non_snake_case)]
 fn SIGRTMAX() -> c_int {
     unsafe { __libc_current_sigrtmax() }
 }
 
-/// Verifies that a signal number is valid: for VCPU signals, it needs to be enclosed within the OS
-/// limits for realtime signals, and the remaining ones need to be between the minimum (SIGHUP) and
-/// maximum (SIGSYS) values.
+/// Verify that a signal number is valid.
+///
+/// For VCPU signal, it needs to be enclosed within the OS
+/// limits for realtime signals. The remaining ones need
+/// to be between the minimum (SIGHUP) and maximum (SIGSYS) values.
```

Comment:
```suggestion
/// to be between the minimum (`SIGHUP`) and maximum (`SIGSYS`) values.
```

Reply:
Fixed

Reply:
Thanks for your review. Addressed the comments with updated PR.